plugins/contactpressuresensorsarray/include/gazebo/ContactPressureSensorsArray.hh

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#ifndef GAZEBOYARP_CONTACTPRESSURESENSORSARRAY_HH
+#define GAZEBOYARP_CONTACTPRESSURESENSORSARRAY_HH
+
+#include <gazebo/common/Plugin.hh>
+#include <string>
+#include <yarp/dev/PolyDriverList.h>
+
+#include <sdf/sdf_config.h>
+#include <sdf/Element.hh>
+#include <sdf/Param.hh>
+
+
+namespace sdf 
+{
+    class Element;
+}
+
+namespace yarp 
+{
+    namespace dev
+    {
+        class IMultipleWrapper;
+    }
+}
+
+namespace gazebo
+{
+    class GazeboYarpContactPressureSensorsArray : public ModelPlugin
+    {
+        public:
+        GazeboYarpContactPressureSensorsArray();
+        virtual ~GazeboYarpContactPressureSensorsArray();
+    
+       /**
+        * Saves the gazebo pointer, creates the device driver
+        */
+        void Load(physics::ModelPtr _parent, sdf::ElementPtr _sdf);
+    
+        private:
+       
+       /**
+        * Polydriver instance to connect to analaog sensor server
+        */
+        yarp::dev::PolyDriver m_dev_wrapper;
```

Comment:
Not a big problem, but I would keep the naming style of the attributes consistent: either `m_devWrapper` or `m_dev_wrapper` , without changing across different attributes.